packages/actor-scraper/playwright-scraper/INPUT_SCHEMA.json

```diff
@@ -0,0 +1,277 @@
+            "default": true
+        },
+        "maxRequestRetries": {
+            "title": "Max request retries",
+            "type": "integer",
+            "description": "Maximum number of times the request for the page will be retried in case of an error. Setting it to 0 means that the request will be attempted once and will not be retried if it fails.",
+            "minimum": 0,
+            "default": 3,
+            "unit": "retries"
+        },
+        "maxPagesPerCrawl": {
+            "title": "Max pages per run",
+            "type": "integer",
+            "description": "Maximum number of pages that the scraper will open. 0 means unlimited.",
+            "minimum": 0,
+            "default": 0,
+            "unit": "pages"
+        },
+        "maxResultsPerCrawl": {
+            "title": "Max result records",
+            "type": "integer",
+            "description": "Maximum number of results that will be saved to dataset. The scraper will terminate afterwards. 0 means unlimited.",
+            "minimum": 0,
+            "default": 0,
+            "unit": "results"
+        },
+        "maxCrawlingDepth": {
+            "title": "Max crawling depth",
+            "type": "integer",
+            "description": "Defines how many links away from the StartURLs will the scraper descend. 0 means unlimited.",
+            "minimum": 0,
+            "default": 0
+        },
+        "maxConcurrency": {
+            "title": "Max concurrency",
+            "type": "integer",
+            "description": "Defines how many pages can be processed by the scraper in parallel. The scraper automatically increases and decreases concurrency based on available system resources. Use this option to set a hard limit.",
+            "minimum": 1,
+            "default": 50
+        },
+        "pageLoadTimeoutSecs": {
+            "title": "Page load timeout",
+            "type": "integer",
+            "description": "Maximum time the scraper will allow a web page to load in seconds.",
+            "minimum": 1,
+            "default": 60,
+            "unit": "secs"
+        },
+        "pageFunctionTimeoutSecs": {
+            "title": "Page function timeout",
+            "type": "integer",
+            "description": "Maximum time the scraper will wait for the page function to execute in seconds.",
+            "minimum": 1,
+            "default": 60,
+            "unit": "secs"
+        },
+        "waitUntil": {
+            "title": "Navigation wait until",
+            "type": "array",
+            "description": "The scraper will wait until the selected events are triggered in the page before executing the page function. Available events are <code>domcontentloaded</code>, <code>load</code> and <code>networkidle</code> <a href=\"https://playwright.dev/docs/api/class-page#page-goto-option-wait-until\" target=\"_blank\">See Playwright docs</a>.",
```

Comment:
`networkidle2` (default prefill) is not listed here. Btw could this be a dropdown? Idk the input schema that well though :)

Reply:
Playwright does not support networkidle2, it has only networkodle, load, and domcontentloaded 🙂

Reply:
yes, its not supported, and you still have it as the default/prefill

<img width="226" alt="image" src="https://user-images.githubusercontent.com/615580/185346234-71dfec7a-eb36-4a8a-9903-978be23cdb33.png">

Reply:
Oh damn 🤦‍♂️

Reply:
Fixed, as for the dropdown, actually yes, puppeteer could have an array of events there, while playwright support only 1 - will update.